AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Support/Utils/AWSCognitoAuthPluginUserDefaults.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct AWSCognitoAuthPluginUserDefaults {
+
+    private static let preferPrivateSessionKey = "privateSessionKey"
```

Comment:
Prefer a more granular namespace: "AWSCognitoAuthPluginUserDefaults.privateSessionKey" or similar. While admittedly unlikely, "privateSessionKey" runs a higher risk of colliding with some other library or app logic.